doc/source/python_reference.rst

```diff
@@ -139,7 +139,7 @@ Functions:
    read_bem_solution
    read_bem_surfaces
    read_cov
-   read_dip
+   read_dipoles
```

Comment:
I agree we should rename but we should change the API too. We should create a Dipole class with a plot method. That will fix our problem with @yousrabk on the RAP-MUSIC code which could then simply return a list of dipole instances with orientation information. 

cc @joewalter